global.json

```diff
@@ -3,7 +3,7 @@
     "Microsoft.Build.Traversal": "1.0.45"
   },
   "sdk": {
-    "version": "3.1.301",
+    "version": "5.0.100",
```

Comment:
Why you jump the version from 3 to 5? Does "5.0.100" from the swagger updates?

Reply:
This is updated in the compute-2020-12 branch